notebooks/Image_Analysis.ipynb

```diff
@@ -109,6 +110,58 @@
     "\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "7b118a48",
+   "metadata": {},
+   "source": [
+    "#### Box Plot Code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "19f35cb9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "box_plot_data=[bg_vars_a, bg_vars_b]\n",
```

Comment:
Should be generalized for accepting more than two datasets.

Reply:
This can be done easily by adding more cell and basically doing the same things as this cell.

Reply:
Can this be automatically done as opposed to doing cell editing?